Makefile

```diff
@@ -20,7 +20,7 @@ test: deps common-test
 test-short: deps common-test-short
 
 .PHONY: generate-go-collector-test-files
-VERSIONS := 1.17 1.18 1.19
+VERSIONS := 1.17 1.18 1.19 1.20
```

Comment:
Hmm. There is a different to not supporting Go 1.17 (as is: It might work, but we don't guarantee) vs actively breaking 1.17. WDYT? I think we don't need to break it actively.